quickwit/quickwit-cli/src/split.rs

```diff
@@ -313,21 +313,28 @@ async fn list_split_cli(args: ListSplitArgs) -> anyhow::Result<()> {
     let metastore = metastore_uri_resolver
         .resolve(&quickwit_config.metastore_uri)
         .await?;
-    let splits = metastore.list_all_splits(&args.index_id).await?;
-
-    let filtered_splits = filter_splits(
-        splits,
-        args.split_states,
-        args.start_date.map(OffsetDateTime::unix_timestamp),
-        args.end_date.map(OffsetDateTime::unix_timestamp),
-        args.create_date.map(OffsetDateTime::unix_timestamp),
```

Comment:
The create date and start_date are the wrong way around, I've swapped these around so they're the correct behaviour when using the new filter.